### PR TITLE
as member photo

### LINKED ISSUE #286: Add member image for Arjun Sivakumar

### PR DESCRIPTION
**Pull request by ctc-devops**
Closes #286

### CODE REVIEW

File: website/src/app/views/our-team/members.js [hunk 1]
```diff
@@ -209,6 +210,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/kei-asakawa/',
         imageURL: kei,
       },
+      {
+        name: 'Arjun Sivakumar',
```
Comment (alanchangxyz) [inline website/src/app/views/our-team/members.js:214]: members should be in alphabetical order after leads (a then k)

**Review by alanchangxyz — CHANGES_REQUESTED** [review]
needs one small change before merging (and if something else gets merged before you finish you may have to fix a merge conflict with `2022-dev-bootcamp-task`), otherwise gj

**Review by claudeisakeeb — CHANGES_REQUESTED** [review]
Good work, same comment as alanchangxyz

**Comment by alanchangxyz** [comment]
Update - another CDS dev got merged in around this spot, you might want to merge in the `2022-dev-bootcamp-task` branch to make sure you're putting your name in the right order before requesting a review 👍

**Pull request closed without merging**